src/info/info.html

```diff
@@ -0,0 +1,19 @@
+<template>
+
+    <div class="row">
+        <template repeat.for="column of columns">
+            <div class="col-sm-6 text-weight-bold">${column.label}</div>
+            <div class="col-sm-6">${column.value}</div>
+        </template>
+    </div>
+    <div class="row">
+        <div class="col-sm-6 text-weight-bold">Channels:</div>
+        <div class="col-sm-6">
+            <template repeat.for="channel of image_info.channels">
+                <span if.bind="!$last">${channel.label},</span>
```

Comment:
There is one other thing on the side: I is recommended that the repeat.for be on the element that it repeats. I noticed you use an extra <template> whereas you can/should put the repeat on the <span>.

I know I have used template in the regions table, the reason being that I have double nesting in that repeat and I cannot produce the html I need in such a case so that I have to rely on this crutch of using <template> as a construct to be able to loop, yet not produce an actual html element. Essentially <template> is a pseudo element that does not reflect in the produced html, however most of the time you want the iteration such that an element is repeatedly produced in which case the repear.for should be on the element.